_includes/messages/server_queries.md

```diff
@@ -97,6 +97,23 @@ Command Examples:
       CONNECT  eff.org 12765 csd.bu.edu
       ; Attempt to connect csu.bu.edu to eff.org on port 12765
 
+### LINKS message
+
+         Command: LUSERS
+      Parameters: None
+
+With LINKS, a user can list all servers which are known by the server answering the query, usually including the server itself.
+
+In replying to the LINKS message, a server MUST send replies back using 0 or more {% numeric RPL_LINKS %} messages and mark the end of the list using a {% numeric RPL_ENDOFLINKS %} message.
+
+Servers MAY omit some or all servers on the network, including itself.
+
+Numeric Replies:
+
+* {% numeric ERR_NOSUCHCOMMAND %}
```

Comment:
```suggestion

```

Reply:
ERR_UNKNOWNCOMMAND is implied for all optional messages in the docs i believe

Reply:
any example?

Reply:
[Text in docs](https://github.com/ircdocs/modern-irc/blob/master/_includes/messages/optional.md?plain=1#L1), i grepped the docs and the reply is never explicitly mentioned below messages. Perhaps the linked note should be moved up as it now only refers to optional messages but in reality should apply to all?

Reply:
That's the "Optional messages" section (which is actually more of a "misc stuff" section). LINKS isn't in that section, because it makes more sense in "Server queries"

Reply:
On second thought, let's move it to that section